plugins/modules/nautobot_server.py

```diff
@@ -0,0 +1,356 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2021, Network to Code <opensource@networktocode.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
```

Comment:
will check

Reply:
it's needed for Python2.7 support, all the other modules still have it

Reply:
The other new modules are pulling this part out since it is a new module, so 3.x is required. I'd be OK removing. Keeping it is kind of nice to have it for posterity.

Reply:
ok, will remove then , not interested in posterity 😆

Reply:
We can always ignore these within the sanity tests, but Ansible sanity tests look for these.

Reply:
thanks, I would leave it there, and we could sanitize it when we decide to do it to all the modules